ckan/controllers/user.py

```diff
@@ -379,7 +379,7 @@ def _save_edit(self, id, context):
             abort(400, _(u'Integrity Error'))
         except ValidationError, e:
             errors = e.error_dict
-            error_summary = e.error_summary
+            error_summary = {_('Password'): _('invalid new password')}
```

Comment:
Hi Karen, I just realize this may break other validation error summary, maybe do not merge it for now. I will have a think. Thank you.